app/assets/javascripts/components/timeline/timeline.jsx

```diff
@@ -181,6 +181,21 @@ const Timeline = createReactClass({
   },
 
   render() {
+    const blocks = document.querySelectorAll('.block__block-content');
```

Comment:
Yeah, I think I'd prefer the other method that just applies going forward, rather than a using JS to insert the styling.